manifests/init.pp

```diff
@@ -29,8 +30,10 @@
   }
 
   file { $confdir:
-    ensure => directory,
-    mode   => '0755',
+    ensure  => directory,
+    mode    => '0755',
+    recurse => $purge_confdir,
```

Comment:
It's there only because the Type Reference specifically says that purge => true only really makes sense with recurse => true.  I didn't really think about it past that.  What is the concern?